ubuntu-agent-vm/scripts/netcoresdk3-1.sh

```diff
@@ -7,3 +7,5 @@ apt-get update
 apt-get install -y apt-transport-https
 apt-get update
 apt-get install -y dotnet-sdk-3.1
+
+export DOTNET_CLI_HOME="/tmp/DOTNET_CLI_HOME"
```

Comment:
what does this change do?

Reply:
In theory it sets the environment variable DOTNET_CLI_HOME

which then lets us not have the following:

The user's home directory could not be determined. Set the 'DOTNET_CLI_HOME' environment variable to specify the directory to use. 
when running dotnet test on the linux agents